Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    fastlane-plugin-wpmreleasetoolkit (3.0.0)
+    fastlane-plugin-wpmreleasetoolkit (3.1.0)
```

Comment:
I noticed this because Peril told me [here](https://github.com/wordpress-mobile/release-toolkit/pull/342#issuecomment-1054957608)
<img width="890" alt="Screen Shot 2022-03-01 at 3 39 48 pm" src="https://user-images.githubusercontent.com/1218433/156105931-2db7353a-f009-41de-a05e-36f086baa832.png">